redirects/17.0.txt

```diff
@@ -75,3 +75,7 @@ applications/sales/point_of_sale/pricing/digital_labels.rst applications/sales/p
 
 applications/services/helpdesk/overview/getting_started.rst applications/services/helpdesk.rst                                                            #helpdesk/overview/getting_started --> helpdesk
 applications/services/project/tasks/email_alias.rst applications/services/project/tasks/task_creation.rst                                                 #project/tasks/email_alias --> task_creation
+
+# applications/website
+
+applications/websites/ecommerce/managing_products/products.rst applications/websites/ecommerce/products.rst
```

Comment:
Sorry if I wasn't clear earlier: You need to add one for each of the sub-pages as well (catalog, variants, etc.)